src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/Tabs.js

```diff
@@ -326,70 +314,72 @@ class Tabs extends React.Component<Props> {
                     index={index}
                     key={index}
                     onClick={this.handleCollapsedTabClick}
-                    skin={tab.props.skin || skin}
```

Comment:
Shouldn't that be `skin={skin}` instead? Or should the collapsed tabs list be always shown in the default skin?

Reply:
The `CollapsedTab` component doesn't have a `skin` property, so that was a bug.